types/leaflet/index.d.ts

```diff
@@ -2045,3 +2583,5 @@ export const extend: typeof Util['extend'];
 export const bind: typeof Util['bind'];
 export const stamp: typeof Util['stamp'];
 export const setOptions: typeof Util['setOptions'];
+
+export function noConflict(): any;
```

Comment:
Creating a global `L` object would require a complete rewrite of leaflet types, and I think it would be too disruptive.
(`export as namespace L ` only creates an object when leaflet is not used as a module)

Reply:
`export as namespace L` allows a module (with exports and imports) to be used in a global context as the namespace L.
As far as I can tell, you'd have to give up on exports and imports in order to create a `namespace L` yourself, and I don't think it's a good direction in addition to being lots of work.